src/components/picker/examples/picker-empty-suggestions.tsx

```diff
@@ -56,8 +59,11 @@ export class PickerExample {
             if (query === '') {
                 // Simulate some network delay
                 setTimeout(() => {
-                    resolve([]);
+                    const result = this.allItems.slice(8, 10);
+                    resolve(result);
                 }, NETWORK_DELAY);
+
+                return;
```

Comment:
We could probably improve this as well if we want to get rid of the `new Promise()` syntax, similar to the other comment I had. The simulated network delay can be done something like this
```ts
await new Promise(resolve => setTimeout(resolve, NETWORK_DELAY));
```

Reply:
⚡ 4d4e4e46b5dd2f45eb94a7f6516a5cf21c47a363